metadata-ingestion/src/datahub/ingestion/source/unity/tag_entities.py

```diff
@@ -29,9 +31,16 @@ class UnityCatalogTagSyncContext(BaseModel):
 logger = logging.getLogger(__name__)
 
 
-class UnityCatalogTagPlatformResourceId(BaseModel, ExternalEntityId):
+def get_unity_catalog_tag_cache_info(
+    platform_resource_repository: "UnityCatalogPlatformResourceRepository",
+) -> Dict[str, Dict[str, int]]:
+    """Get cache statistics for Unity Catalog tag operations."""
+    return platform_resource_repository.get_entity_cache_info()
```

Comment:
the `get_unity_catalog_tag_cache_info` method feels like an extra and unnecessary layer of indirection